.github/actions/build-deps/action.yml

```diff
@@ -42,12 +49,30 @@ runs:
           !${{ inputs.workingDirectory }}/*_build_temp/*.tar.gz
           !${{ inputs.workingDirectory }}/*_build_temp/*.tar.xz
           !${{ inputs.workingDirectory }}/*_build_temp/*.zip
+          !${{ inputs.workingDirectory }}/*_build_temp/qt*
```

Comment:
Why is Qt being added here?

Reply:
Probably a remnant of an older build environment on CI.